stock_valuation_layer_accounting_date/tests/test_stock_valuation_layer_accounting_date.py

```diff
@@ -38,5 +39,6 @@ def test_svl_accounting_date_manual_periodic(self):
         self._make_in_move(self.product1, 10)
         valuation_layer = self.product1.stock_valuation_layer_ids
         self.assertEqual(
-            valuation_layer.accounting_date, valuation_layer.create_date.date()
+            valuation_layer.accounting_date,
+            fields.Date.context_today(valuation_layer, valuation_layer.create_date),
```

Comment:
```suggestion
            fields.Date.context_today(self, valuation_layer.create_date),
```
I guess it doesn't change the result, but isn't it more appropriate this way?

Reply:
Test class object doesn't have  no attribute '_context' and we can use `self` in here.